lib/shelley/bench/Restore.hs

```diff
@@ -313,7 +313,8 @@ cardanoRestoreBench tr c socketFile = do
         , benchRestoreSeqWithOwnership (Proxy @4)
 
         , benchRestoreRndWithOwnership (Proxy @0)
-        , benchRestoreRndWithOwnership (Proxy @1)
+        -- benchRestoreRndWithOwnership (Proxy @1)
+        -- Removed to prevent timeout in CI.
```

Comment:
Keeping it commented to prevent someone from thinking the absence of it is a mistake.

And I kept 0.4% rnd, because I recall it being similar to some exchanges (but I don't really know).